typegraph/deno/sdk/src/runtimes/kv.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+// SPDX-License-Identifier: MPL-2.0
+
+import { Materializer, Runtime } from "./mod.ts";
+import * as t from "../types.ts";
+import { runtimes } from "../wit.ts";
+import { Effect, KvMaterializer } from "../gen/typegraph_core.d.ts";
+
+import { fx } from "../index.ts";
+
+class KvOperationMat implements Materializer {
+  operation: KvMaterializer;
+  _id: number;
+  constructor(id: number, operation: KvMaterializer) {
+    this._id = id;
+    this.operation = operation;
+  }
+}
+
+export class KvRuntime extends Runtime {
+  host: string;
+  port?: string;
+  dbNumber?: number;
+  password?: string;
+
+  constructor(
+    { host, port, dbNumber, password }: {
```

Comment:
Can we combine all these values in a single field (URL) like we have for the `PrismaRuntime`? for consistency...